english/your-dog/your-dog.md

```diff
@@ -1,3 +1,5 @@
 Then you wake up! Or did you really? But your dog is licking you in the face. Lord, that dog breath!
 
+An evil vile smell of rotten, no desicated, dry dog food that would haunt an arch demon used to the hellish smell of sulfur fire burning flesh.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.